kafka-connect-gcp-storage/src/main/scala/io/lenses/streamreactor/connect/gcp/storage/config/CommonConfigDef.scala

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2017-2023 Lenses.io Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.lenses.streamreactor.connect.gcp.storage.config
+
+import io.lenses.streamreactor.connect.cloud.common.config.CompressionCodecConfigKeys
+import GCPConfigSettings.ERROR_POLICY
+import GCPConfigSettings.ERROR_POLICY_DEFAULT
+import GCPConfigSettings.ERROR_POLICY_DOC
+import GCPConfigSettings.ERROR_RETRY_INTERVAL
+import GCPConfigSettings.ERROR_RETRY_INTERVAL_DEFAULT
+import GCPConfigSettings.ERROR_RETRY_INTERVAL_DOC
+import GCPConfigSettings.GCP_PROJECT_ID
+import GCPConfigSettings.GCP_QUOTA_PROJECT_ID
+import GCPConfigSettings.HOST
+import GCPConfigSettings.HTTP_CONNECTION_TIMEOUT
+import GCPConfigSettings.HTTP_CONNECTION_TIMEOUT_DEFAULT
+import GCPConfigSettings.HTTP_CONNECTION_TIMEOUT_DOC
+import GCPConfigSettings.HTTP_NBR_OF_RETIRES_DEFAULT
+import GCPConfigSettings.HTTP_NBR_OF_RETRIES
+import GCPConfigSettings.HTTP_NBR_OF_RETRIES_DOC
+import GCPConfigSettings.HTTP_SOCKET_TIMEOUT
+import GCPConfigSettings.HTTP_SOCKET_TIMEOUT_DEFAULT
+import GCPConfigSettings.HTTP_SOCKET_TIMEOUT_DOC
+import GCPConfigSettings.KCQL_CONFIG
+import GCPConfigSettings.KCQL_DOC
+import GCPConfigSettings.NBR_OF_RETIRES_DEFAULT
+import GCPConfigSettings.NBR_OF_RETRIES
+import GCPConfigSettings.NBR_OF_RETRIES_DOC
+import org.apache.kafka.common.config.ConfigDef
+import org.apache.kafka.common.config.ConfigDef.Importance
+import org.apache.kafka.common.config.ConfigDef.Type
+
+trait CommonConfigDef extends CompressionCodecConfigKeys with AuthModeSettingsConfigKeys {
+
+  def config: ConfigDef = {
+    val conf = new ConfigDef()
+      .define(
+        GCP_PROJECT_ID,
+        Type.STRING,
+        "",
+        Importance.HIGH,
+        "Azure Account Name",
```

Comment:
We have a copy-paste issue here. It should say GCP. It applies to the other properties